packages/fuchsia_remote_debug_protocol/lib/src/fuchsia_remote_connection.dart

```diff
@@ -402,15 +404,18 @@ class FuchsiaRemoteConnection {
     return result;
   }
 
-  Uri _getDartVmUri(int port) {
+  Uri _getDartVmUri(PortForwarder pf) {
     // While the IPv4 loopback can be used for the initial port forwarding
```

Comment:
Does this comment need an update?

Reply:
This comment is still accurate, but it's specific to the SSH port forwarder. Merged it with the comment in _SshPortForwarder.start.